linux/runtime.go

```diff
@@ -204,6 +207,7 @@ func (r *Runtime) forward(events shim.Shim_EventsClient) {
 			Pid:        e.Pid,
 			ID:         e.ID,
 			ExitStatus: e.ExitStatus,
+			ExitedAt:   e.ExitedAt,
```

Comment:
Ahh, I see.  I'll add a timestamp to the event object in a separate PR and make sure that it is propagated to the external API.